content/pages/devices/dummyMRT.rst

```diff
@@ -0,0 +1,5 @@
+7T Scanner
```

Comment:
s/7T/?T/

Reply:
Don't yet - the page is just a placeholder to show where things belong. We need to figure out the correct list of devices and how they differ (at least location) to see what we actually need.

Reply:
Ahhh... sorry, I thought you were going to keep this in place as a demo/dummy page, in which case I was suggesting that it not have any definite data in it, and that a ? would be a better fit.

I didn't realize this is an in-progress page that will be renamed and gain content.